homeassistant/components/light/greenwave.py

```diff
@@ -0,0 +1,108 @@
+"""
+Support for Greenwave Reality (TCP Connected) lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.tcpbulbs/
+"""
+import logging
+import voluptuous as vol
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, Light, PLATFORM_SCHEMA, SUPPORT_BRIGHTNESS)
+from homeassistant.const import CONF_HOST
+import homeassistant.helpers.config_validation as cv
+
+SUPPORTED_FEATURES = (SUPPORT_BRIGHTNESS)
+REQUIREMENTS = ['greenwavereality==0.2.9']
+_LOGGER = logging.getLogger(__name__)
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required("version"): cv.positive_int,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup TCP Connected Platform."""
+    import greenwavereality as greenwave
+    import os
+    host = config.get(CONF_HOST)
+    tokenfile = hass.config.path('greenwave.token')
```

Comment:
use `.greenwave` so that should be not editable for users with a `.`

Reply:
Changing that now